src/GLM.jl

```diff
@@ -9,6 +9,9 @@ module GLM
     using StatsBase: StatsBase, CoefTable, StatisticalModel, RegressionModel
     using StatsFuns: logit, logistic
     using Distributions: sqrt2, sqrt2π
+    if VERSION >= v"0.6"
```

Comment:
Never mind, that's not what I was thinking of. Yeah I have no idea why that's happening because SpecialFunctions shouldn't be exporting anything on 0.5. Good that `import` can get around it.